struct.go

```diff
@@ -234,7 +235,7 @@ func rSlice(ra *rand.Rand, t reflect.Type, v reflect.Value, tag string, size int
 	return nil
 }
 
-func rString(ra *rand.Rand, t reflect.Type, v reflect.Value, tag string) error {
+func rString(ra *rand.Rand, _ reflect.Type, v reflect.Value, tag string) error {
```

Comment:
If reflect type isnt being use you can just remove it in the switch statement call and not need to pass it.